kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/record/RecordTestUtils.java

```diff
@@ -71,7 +81,7 @@ public static String recordKeyAsString(Record record) {
      */
     public static Record record(byte[] value,
```

Comment:
not necessarily for now, but "record" (in method names and formal parameters) is conflicting with Java keyword.

Reply:
It's not a keyword, it's a restricted identifier. Personally I remain to be convinced that these sonar warnings have any value, or that fixing them is not a waste of effort. Can anyone think of a realistic usage where someone reading the code it going to get confused by the fact that `record` means one thing in a type declaration and something else when used as a variable name?